### ISSUE TITLE
Add feed properties to PublishNodeJsModule task

### ISSUE THREAD

**Issue by splatteredbits** [webmd-health-services/Whiskey#8]
Add `RegistryUri`, `CredentialID`, and `Email` properties to the `PublishNodeJsModule` task to replace these properties on the `ProGetSession` property of the task context.

```
PublishTasks:
- PublishNodeJsModule:
    RegistryUri: https://npm.example.com/
    CredentialID: NpmCredential
    Email: user@example.com
```

**Comment by scott-pickthorn** [comment]
only registryUri is done. Credentials and email still needs to be done.

**Comment by splatteredbits** [comment]
Resolved by 80a4c5956771d7935aecf738980ad16703c05472.
